assets/js/phoenix_live_view.js

```diff
@@ -2095,6 +2095,10 @@ export class View {
   }
 
   withinTargets(phxTarget, callback){
+    if(phxTarget.nodeType) {
```

Comment:
Is this the right check here?

I was thinking this would look like `if (phxTarget instanceof HTMLElement)`

Reply:
Not a JS master here, so I was not sure how to do that the best way. Checked here and you're right, node elements are instances of HTMLElement, will change.